CHANGELOG.md

```diff
@@ -3,6 +3,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [0.0.1] - TBD
+## [1.1.0] - TBD
 ### Added
-- Docker Image for Shunting Yard
+- An option (`--ct-config`) to pass Graphite configuration to Shunting Yard.
```

Comment:
Graphite configuration -> common Circus Train configuration
(since now it won't be limited to just the Graphite section right?)

Reply:
True. changed.